### PR TITLE
Add REPL.it snippets to Java articles

### PR DESCRIPTION
**Pull request by koustuvsinha**
* Added [REPL.it](https://repl.it) snippets wherever applicable
* Fixed code samples

@alayek have a look.

### CODE REVIEW

**Comment by Rafase282** [comment]
Tested some relp, things look good. I'l let @alayek take a more in-depth look.

File: Java-Loops.md [hunk 1]
```diff
@@ -89,7 +93,8 @@ for (initialization; expression; increment)
 * `increment` - Invoked after each iteration through the loop. You can increase/decrease the value of variables here.
 
 ```java
-for (int iter_For = 0; iter_For < 10; iter_For++) 
+int iter_For;
```
Comment (alayek) [inline Java-Loops.md]: Why this change? The Java Compiler would do this optimization anyway.

Comment (koustuvsinha) [inline Java-Loops.md]: its because to display the value of iterFor after the loop. check the Repl code

**Comment by alayek** [comment]
# LGTM :+1: 

I have tested the code samples on REPL.

@abhisekp @theflametrooper just check once in case I missed something. Will merge ASAP.

File: Java-Loops.md (hunk 1 above)
Comment (alayek) [inline Java-Loops.md]: Ok cool! Got it thanks.

**Comment by theflametrooper** [comment]
## LGTM :+1:
